### DISCUSSION TITLE
Errors when runing interface

### DISCUSSION THREAD

**Discussion by liod1234** [fredzzhang/upt#70] [Q&A] [answered]
Hi, I am new to HOI detectors and I try to run this code. However, when I try to run
`python inference.py --resume checkpoints/upt-r50-hicodet.pt --index 8789`
it keeps give me error as follows:
`=> Start from a randomly initialised model
Traceback (most recent call last):
  File "inference.py", line 225, in <module>
    main(args)
  File "/Users/leoliu/opt/miniconda3/envs/pocket/lib/python3.8/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "inference.py", line 172, in main
    visualise_entire_image(image, output[0], actions, args.action, args.action_score_thresh)
IndexError: list index out of range`
Could you help me to fixed this and tell me how to use your code? Thanks very much.

**Comment by fredzzhang** [comment, marked as answer]
Hi @liod1234,

The output `=> Start from a randomly initialised model` suggests the specified checkpoint path does not exist. Have you downloaded the checkpoint to the correct directory?

Fred.

**Reply by liod1234** [reply]
Yes, u r right. I have fixed it. Thanks very much.
